class/voyage.class.php

```diff
@@ -240,14 +176,74 @@ public function __construct(DoliDB $db)
 	 */
 	public function create(User $user, $notrigger = false)
 	{
+        global $conf;
+
+        $this->setTarifByTag();
+
 		$resultcreate = $this->createCommon($user, $notrigger);
 
-		//$resultvalidate = $this->validate($user, $notrigger);
+
+        $this->add_object_linked('product', $this->array_options['options_product'], $user, $notrigger);
 
 		return $resultcreate;
 	}
 
-	/**
+    /**
+     * En fonction du tag associé à un voyage, un prix lui est associé
+     * @return int|void
```

Comment:
Le return de la php doc pas bon, préciser que la fonction retourne toujours in et 1 si OK, -1 si KO